tests/unit/has.ts

```diff
@@ -96,6 +96,79 @@ registerSuite({
 				hasAdd('one', true, true);
 				assert.isTrue(has('one'));
 			}
+		},
+
+		'has loader tests': {
```

Comment:
I haven't been able to get these tests passing using `grunt test`. I'm seeing errors like:

```
Error: Attempt to require unloaded module src/has!intern!object
  at contextRequire  <loader.ts:385:12>
  at module.require.result  <loader.ts:416:11>
  at Test.registerSuite.has loader tests.both feature and no-feature modules provided [as test]  <_build/tests/unit/has.js:100:35>
```

This is probably because the single argument require() expects the module to already be loaded.

I'm not certain we want to conflate testing has.normalize with require. Why don't we just bring in `normalize` and `load` and test these methods directly.

```
import { normalize as hasNormalize, load as hasLoad } from 'src/has';
```

This will verify the unit under test without testing the loader's require statement :)

```
'test'() {
	var expected = has('host-browser') ? 'intern' : 'intern!object';
	var actual = hasNormalize('src/has!host-browser?intern:intern!object', (<any> require).toAbsMid);
	assert.strictEqual(actual, expected);
},
```